wcst_encode/data_utils.py

```diff
@@ -0,0 +1,128 @@
+from .constants import FEATURES
+import numpy as np
+import pandas as pd
+
+def get_behavior_by_bins(bin_size, beh):
+    """
+    bin_size: in miliseconds, bin size
+    data: dataframe for behavioral data from object features csv
+    Returns: new dataframe with one-hot encoding of features, feedback
+    """
+    max_time = np.max(beh["TrialEnd"].values)
+    max_bin_idx = int(max_time / bin_size) + 1
+    columns = FEATURES + ["CORRECT", "INCORRECT"]
+    types = ["f4" for _ in columns]
+    zipped = list(zip(columns, types))
+    dtype = np.dtype(zipped)
+    arr = np.zeros((max_bin_idx), dtype=dtype)
+
+    for _, row in beh.iterrows():
+        # grab features of item chosen
+        item_chosen = int(row["ItemChosen"])
+        color = row[f"Item{item_chosen}Color"]
+        shape = row[f"Item{item_chosen}Shape"]
+        pattern = row[f"Item{item_chosen}Pattern"]
+
+        chosen_time = row["FeedbackOnset"] - 800
+        chosen_bin = int(chosen_time / bin_size)
```

Comment:
generally best to be explicit with things like `np.ceil` and `np.floor` (here it is `np.floor`)